hack/fetch-releases.sh

```diff
@@ -120,7 +120,48 @@ release_yaml_pac() {
     echo ""
 }
 
-#Args: <target-platform> <pipelines version> <triggers version> <dashboard version> <results version> <pac version>
+
+release_yaml_hub() {
+  echo fetching '|' component: ${1} '|' version: ${2}
+  local version=$2
+
+  ko_data=${SCRIPT_DIR}/cmd/${TARGET}/operator/kodata
+  dirPath=${ko_data}/tekton-hub/${version}
+  rm -rf ${dirPath} || true
+  mkdir -p ${dirPath} || true
+
+  url=""
+  components="db db-migration"
+
+  if [[ ${TARGET} == "openshift" ]]; then
+    components="$components api-openshift"
+  else
+    components="$components api-k8s"
+  fi
+
+  dirNames=(db db-migration api)
+  arrayIndex=0
```

Comment:
```suggestion
  components="db db-migration api"
```

and then inside the for loop we can add an if condition specifically for api.
for example

```bash
  fileName=${component}.yaml
  [[ ${component} == "api" ]] && fileName=${component}-${TARGET}.yaml
```

and dest can be `dest=${dirPath}/${component}`

we will not need `arrayIndex=$((arrayIndex+1))`